src/client/auth.rs

```diff
@@ -389,11 +415,16 @@ pub struct Credential {
     /// Additional properties for the given mechanism.
     pub mechanism_properties: Option<Document>,
 
-    /// The token callbacks for OIDC authentication.
+    /// The token callback for OIDC authentication.
     /// TODO RUST-1497: make this `pub`
+    /// Credential::builder().oidc_callback(oidc::Callback::human(...)).build()
```

Comment:
The reason for having the `oidc::State` named `oidc_callback` is to make this more clear for users, so they will build this with code like the above. What do you think?

Reply:
The nice thing about doing this this way is that it is automatically impossible to add both human and machine call backs, so we do not even need an error for that